source/_components/pioneer.markdown

```diff
@@ -40,9 +44,84 @@ timeout:
   description: Number of seconds (float) to wait for blocking operations like connect, write and read.
   required: false
   type: float
+sources:
+  description: A list of mappings from source friendly name to source code (e.g. `TV:'05'`). Valid source codes depend on the receiver (some known codes can be found below). Codes must be defined as strings (between single or double quotation marks), so that `05` is not implicitly transformed to `5`, which wouldn't be a valid source code.
+  required: false
+  default: Empty list (i.e. no soruce selection will be possible)
+  type: list
 {% endconfiguration %}
 
 Notes:
 
 - Some Pioneer AVRs use the port 23 default and some are reported to use 8102.
 - `timeout` is a socket level option and should only be configured if you know what you are doing.
+
+
+### Source codes
+
+Under these lines you can find some sample `sources` lists per receiver model. Here we use the source names as shown on the remote as key for each code. However these are for display purposes only, so you could rename inputs to better match your set-up (e.g. `HDMI: '19'` to `Kodi: '19'`.
+
+Codes must be defined as strings (between single or double quotation marks), so that `05` is not implicitly transformed to `5`, which wouldn't be a valid source code.
```

Comment:
```suggestion
Codes must be defined as strings (between single or double quotation marks) so that `05` is not implicitly transformed to `5`, which wouldn't be valid source code.
```